src/api.js

```diff
@@ -376,3 +376,61 @@ api.declare({
   }
   return res.reply({installed: false});
 });
+
+// Below is the draft code to make the badges clickable.
+// In the finished thing, the helper function will go to the beginning of the module,
+// and will be used in other APIs as well.
+api.declare({
+  name: 'taskLink',
+  title: 'Redirects to the task inspector page',
+  description: [
+    'Builds a link to the task inspector for the given task group',
+    ' and redirects the user to that page.',
```

Comment:
Also, that's not really what this method is doing anymore, since it's just returning status.target_url.